Watchman.Discord/Areas/Help/Services/HelpMessageGeneratorService.cs

```diff
@@ -34,6 +35,13 @@ public string GenerateJsonHelp(IEnumerable<HelpInformation> helpInformations)
             return serialized;
         }
 
+        public string GenerateJsonHelpForOneCommand(string command, IEnumerable<HelpInformation> helpInformations)
+        {
+            var helpInformation = helpInformations.First(x => x.CommandName.ToLower() == command + "command");
```

Comment:
po co szukasz pasującej komendy dwa razy?

Reply:
w sensie? z tego co wiem, to szuka pierwszej komendy z Enuma która sie zgadza, gdzie jest drugie szukanie?

Reply:
Drugie szukanie masz w PrintHelpForOneCommand na początku metody.